docs/src/metabolic-modeling.jl

```diff
@@ -226,12 +226,12 @@ function optimized_vars(cs::C.ConstraintTree, objective::C.LinearValue, optimize
     JuMP.@objective(model, JuMP.MAX_SENSE, C.substitute(objective, x))
     function add_constraint(c::C.Constraint)
         b = c.bound
-        if b isa Float64
-            JuMP.@constraint(model, C.substitute(c.value, x) == b)
-        elseif b isa Tuple{Float64,Float64}
+        if b isa C.EqualTo
+            JuMP.@constraint(model, C.substitute(c.value, x) == b.equal_to)
+        elseif b isa C.Between
```

Comment:
nice

Reply:
technically people could even go for something like an overloaded `constrainize(x::Bound)` but let's not go too far at this point...  maybe in the MILP tutorial.